tasks.c

```diff
@@ -8366,14 +8373,14 @@ static void prvAddCurrentTaskToDelayedList( TickType_t xTicksToWait,
                 /* Wake time has overflowed.  Place this item in the overflow
                  * list. */
                 traceMOVED_TASK_TO_OVERFLOW_DELAYED_LIST();
-                vListInsert( pxOverflowDelayedTaskList, &( pxCurrentTCB->xStateListItem ) );
+                vListInsert( pxOverflowDelayedList, &( pxCurrentTCB->xStateListItem ) );
```

Comment:
`prvAddCurrentTaskToDelayedList` is called with scheduler suspended. Can we consider the following code to reduce the logic?
```
ListItem_t * pxCurrentTaskStateItem = pxCurrentTCB->xStateListItem;

vListInsert( pxDelayedTaskList, pxCurrentTaskStateItem );
vListInsert( pxOverflowDelayedList, pxCurrentTaskStateItem );
```

Reply:
This does not impact the MISRA violation, do you want this in a separate PR?

Reply:
The comment above is suggestion only. The original implementation is okay to me.

Reply:
Ok.